.github/workflows/chainsaw-e2e.yaml

```
@@ -91,8 +91,8 @@ jobs:
     strategy:
       fail-fast: false    
       matrix:
-        k8s-version: [v1.27.3, v1.26.3]
-        n4k-chart-version: [3.1.14]
+        k8s-version: [v1.30.0, v1.29.4, v1.28.9]
```

```suggestion
        k8s-version: [v1.30.0]
```

We use the below configuration to create kind clusters with alpha API enabled for VAPs.
```
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
featureGates:
  ValidatingAdmissionPolicy: true
runtimeConfig:
  admissionregistration.k8s.io/v1alpha1: true
nodes:
  - role: control-plane
  - role: worker
```

We can't use this configuration in versions `1.28` and `1.29`, not sure why. But we would have to remove these versions.

@MariamFahmy98, could you please help explain why we can enable both the alpha and beta APIs, or just the beta API, but not just the alpha API for VAPs in Kubernetes versions `1.28` and `1.29`? This problem doesn't occur in `1.30` though.